chart/gatekeeper-operator/helm-modifications/helm-modifications.yaml

```diff
@@ -53,6 +53,7 @@ spec:
             - --port=8443
             - --logtostderr
             - --constraint-violations-limit={{ .Values.constraintViolationsLimit }}
+            - --audit-from-cache={{ .Values.auditFromCache }}
```

Comment:
We need to update this and other flag fixes to use the `manifest_staging` location for holding pending manifest changes

Reply:
Done